content/de/community/_index.html

```diff
@@ -151,7 +151,7 @@ <h1 style="padding-top:1%">Diskussionen</h1>
 </div>
 
 <div class="resourcebox">
-<img src="/images/community/twitter.png" alt="Twitter" style="width:80%;padding-bottom:2%">
+<img src="static\images\x.png" alt="Twitter" style="width:80%;padding-bottom:2%">
```

Comment:
The absolute path follows '/' forward slashes
```suggestion
<img src="/images/community/x.png" alt="X-org" style="width:80%;padding-bottom:2%">
```

Reply:
i have done my changes

Reply:
Appreciated, But I still see the changes in de (`content/de/`) localization. Make the same changes in en (`content/en`) and revert the changes on de page

Reply:
i have solved the issue

Reply:
> Let's move the image inside the community directory

I would check again, Is the image currently where it's supposed to be?